libs/common/utilities/src/css/vars/colors.var.css.ts

```diff
@@ -0,0 +1,24 @@
+import { CSSResult, css } from 'lit';
+
+// ! Indien je een kleur aanpast, kijk dan even na of er andere kleuren op gebaseerd zijn die een alpha waarde toevoegen.
+
```

Comment:
Eventueel zou je gelijkaardig zoals je deed bij `fonts.css.ts` ook dit kunnen doen:

```ts
const actionColor = "#0055cc";
```

en dan in je CSSResult opzet:

```TS
        --vl-action-color: ${unsafeCSS(actionColor)};
        ...
        --vl-focus-color: ${unsafeCSS(actionColor)}a6;
```

Maar geef toe dat dit wel minder leesbaar is (natuurlijk dat is ook zo bij fonts opzet). Voordeel is wel dat je meer je best moet doen om per ongeluk dingen kapot te maken.

Nadeel ook nog is dan dat de IDE ook niet meer makkelijk het kleur er uit kan halen maar dat is minor mss.

Reply:
Maakt me ook hier niet uit, we zien wel hoe uitgebreid / beheerbaar de globale kleuren worden en passen het dan wel aan als de andere aanpak beter lijkt.